### PR TITLE
bug(ES-298902): provided source code for the azure nodejs file provider

### PR DESCRIPTION
**Pull request by vishwanathan3797**
### Description:

Added the source code for the azure nodejs file provider

### CODE REVIEW

File: filesystem-server.js [hunk 1]
```diff
@@ -0,0 +1,705 @@
...
+app.use(json());
+
+const accountName = "<--Your Account Name-->"; // For Example: "ej2azureblobstorage"
+const accountKey = "<--Your Account Key-->";
+const EndpointSuffix = "<--Your Account Endpoint-->"; // For Example: "core.windows.net"
+const containerName = 'files';
+const directoryName = 'Files';
+const endSlash = '/';
+const connectionString = `DefaultEndpointsProtocol=https;AccountName=${accountName};AccountKey=${accountKey};EndpointSuffix=${EndpointSuffix}`;
+//For store the file in bufffer objects
```
Comment (keerthanaRajendran) [inline filesystem-server.js]: Correct the spelling @vishwanathan3797

File: filesystem-server.js [hunk 2]
```diff
@@ -0,0 +1,705 @@
...
+            let location;
+            let isFile = false;
+            let created;
+            let modified;
+            for (const item of req.body.names) {
+                if (req.body.data[0].isFile) {
+                    const blobClient = containerClient.getBlobClient(directoryName + req.body.path + item);
+                    const properties = await blobClient.getProperties();
+                    names.push(basename(blobClient.name));
+                    // Replace the blobClient.name to get the common loaction for more thatn one files
```
Comment (keerthanaRajendran) [inline filesystem-server.js]: Correct the spelling in comment lines @vishwanathan3797

**Comment by keerthanaRajendran** [comment]
Update a Read me file on how to run this @vishwanathan3797

File: filesystem-server.js (hunk 1 above)
Comment (vishwanathan3797) [reply filesystem-server.js]: spelling corrected

File: filesystem-server.js (hunk 2 above)
Comment (vishwanathan3797) [reply filesystem-server.js]: corrected the spelling

File: README.md [hunk 3]
```diff
@@ -1,2 +1,123 @@
...
+
+- To install the depend packages for the file system provider, navigate to @syncfusion/ej2-filemanager-node-filesystem folder within the node_modules and run the below command 
+
+```sh
+ 
+  npm install
+
+```
+
+### start the service
```
Comment (keerthanaRajendran) [inline README.md]: Start - use caps

File: README.md [hunk 4]
```diff
@@ -1,2 +1,123 @@
 # filemanager-azure-nodejs-file-provider
+
 This repository contains the node JS Azure blob storage file system providers for the Syncfusion File Manager component.
+
+## Key Features
+
+The Node.js file provider module allows you to work with the azure blob storage. It also provides the methods for performing various file actions like creating a new folder, renaming files, and deleting files.
+
+NodeJs File Provider Serves the file system providers support for the  FileManager component with the NodeJS.
```
Comment (keerthanaRajendran) [inline README.md]: Follow same casing NodeJS

File: README.md (hunk 3 above)
Comment (vishwanathan3797) [reply README.md]: changes committed

File: README.md (hunk 4 above)
Comment (vishwanathan3797) [reply README.md]: changes committed

**Review by keerthanaRajendran — APPROVED** [review]

**Pull request merged**